docs/data-structures/postordertraversal.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Postorder traversal of Binary Tree
+slug: /data-structures/postordertraversal
+---
+
+## Introduction
+
+Binary Tree is a type of tree which have at most two children or we can say at most degree 2. So to traverse the binary tree according to Depth-first search. We have a traversal algorithm called Postorder traversal.
+
+## Explanation 
+In this type of traversal(Postorder traversal). first, we have to visit left part then right part then its root respectively. In this first visit left part that means we have to traverse the whole left subtree in this order (left->right->root) after that whole right subtree in this order (left->right->root) and then visit root of that subtree. Postorder: (left->right->root). 
```

Comment:
Make it:

In this type of traversal(Postorder traversal). first, we have to visit the left part then the right part then its root respectively. In this first visit left part that means we have to traverse the whole left subtree in this order (left->right->root) after that whole right subtree in this order (left->right->root) and then visit the root of that subtree. Postorder: (left->right->root).